#154953785 Make translations easier

kevinsamoei opened the pull request:
#### What does this PR do?
It allows a user to filter exercises and Ingredients by language so as to make it easier for them to view them in a specific language.
#### Description of Task to be completed?
Every ingredient and exercise has a field indicating the language, which is automatically set as the user's current language when adding them.
The current list is then filtered based on this.

The user should be able to select which languages to show. They should not be shown only their current language.
#### How should this be manually tested?
Navigate to either the exercises or Ingredients overview then use the filter by language button to get the ingredients/exercises in that specific language.
#### What are the relevant pivotal tracker stories?
[#154953785](https://www.pivotaltracker.com/story/show/154953785)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-02-28 at 12 59 08" src="https://user-images.githubusercontent.com/9350722/36781544-383caab2-1c87-11e8-8de3-bc4a2827b3f8.png">

JoyyToo reviewed (approved):
LGTM

muchai-mercy commented on wger/exercises/templates/exercise/overview.html:
```diff
@@ -8,7 +8,22 @@
+{% trans "Exercises" %}
+<div class="btn-group" style="float:right;">
+    <button type="button" class="btn btn-primary dropdown-toggle btn-xs" data-toggle="dropdown">
+            {% trans "Filter exercises by language" %}  <span class="caret"></span>
```
We can update this button to less words as: `Filter by language`

Sylvance replied:
Yeah, good.

Sylvance committed:
[feature #154953785] Make translations easier:
 - Add translations for ingridients
 - Add translations for exercises
 - Add filter by buttons to the templates
[Delivers #154953785]

muchai-mercy reviewed (approved):
LGTM
